NightscoutServiceKit/NightscoutService.swift

```diff
@@ -337,15 +337,11 @@ extension NightscoutService: RemoteDataService {
             return .failure(NotificationValidationError.missingOTP)
         }
         
```

Comment:
This sent-at date is also used in `DeviceDataManager.handleRemoteNotification` for push age validation. That logic has not changed.  

But I started _also_ using the date here to give better error messaging, which introduced an issue on older NS versions which don't have this date. I'm now making it optional here.